core/src/main/java/jenkins/console/ConsoleUrlProvider.java

```diff
@@ -62,7 +62,7 @@ public interface ConsoleUrlProvider extends Describable<ConsoleUrlProvider> {
      * Get a URL relative to the context path of Jenkins which should be used to link to the console for the specified build.
      * <p>Should only be used in the context of serving an HTTP request.
      * @param run the build
-     * @return the URL for the console for the specified build, relative to the context of Jenkins, or {@code null}
+     * @return the URL for the console for the specified build, relative to the context of Jenkins (should not start with {@code /}), or {@code null}
      * if this implementation does not want to server a special console view for this build.
```

Comment:
It would be nice to fix this typo:

```suggestion
     * if this implementation does not want to serve a special console view for this build.
```